extension/lib/doc.js

```diff
@@ -265,22 +283,27 @@ class Doc {
         return container_sentences_map;
     };
 
-    /*
+        /*
     Returns: string[], given 
```

Comment:
Remove returns statement. Instead, explain that this set this.keywords.

Reply:
👍